src/main/java/com/thealgorithms/maths/AbsoluteMax.java

```diff
@@ -17,7 +17,7 @@ public static int getMaxValue(int... numbers) {
         }
         int absMax = numbers[0];
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (Math.abs(numbers[i]) > Math.abs(absMax) || (Math.abs(numbers[i]) == Math.abs(absMax) && numbers[i] > absMax)) {
                 absMax = numbers[i];
             }
```

Comment:
```suggestion
            if (Math.abs(numbers[i]) >= Math.abs(absMax)) {
                absMax = Math.max(absMax, numbers[i]);
            }
```
Can it be done like that? The initially proposed condition looks quite complex

Reply:
Surely we can do like that, but in term of cost/performance **Math.max()** will be called in every iteration (on matched condition) that's why I preferred to use logical operation (means if Math.abs(numbers[i]) > Math.abs(absMax) will be true **the rest of the condition will be ignored**) and in this way we can eliminate **Math.max()** cost.

Please suggest .!